### PR TITLE
Transactions with a link in the Patron ID field are not included in the list of search results by Patron ID.

### PR DESCRIPTION
**Pull request by Terala-Priyanka**
## Purpose
Transactions with a link in the Patron ID field are not included in the list of search results by Patron ID.

On transaction Detail, in Patron Information Accordion, folioPatronId was displayed. This has been now replaced with patronId. These details can be viewed or referred from API https://volaris-okapi.ci.folio.org/inn-reach/transactions/b2df294b-469d-4f3f-ae6e-9ba332f7922a (just as an example transaction).

## Refs
https://issues.folio.org/browse/UIINREACH-200

## Screenshots
chrome_azOU5qIFiB

K2LBoYUEKS

### CODE REVIEW

**Comment by Terala-Priyanka** [comment]
@folio-org/fe-tl-reviewers please take a look

File: src/components/transaction/TransactionDetails/components/PatronInformation/PatronInformation.test.js [hunk 1]
```diff
@@ -1,11 +1,16 @@
 import React from 'react';
 import { createMemoryHistory } from 'history';
-import { screen } from '@testing-library/react';
+import { screen, fireEvent  } from '@testing-library/react';
 import { renderWithIntl } from '@folio/stripes-data-transfer-components/test/jest/helpers';
 import { Router } from 'react-router';
 import { translationsProperties } from '../../../../../../test/jest/helpers';
 import PatronInformation from './PatronInformation';
 
+// jest.mock('react-router-dom', () => ({
```
Comment (ryandberger) [inline src/components/transaction/TransactionDetails/components/PatronInformation/PatronInformation.test.js:9]: Please remove commented-out code.

**Review by ryandberger — APPROVED** [review]

File: src/components/transaction/TransactionDetails/components/PatronInformation/PatronInformation.test.js (hunk 1 above)
Comment (Terala-Priyanka) [reply src/components/transaction/TransactionDetails/components/PatronInformation/PatronInformation.test.js]: Thank you for the observation. Removed the commented lines.

**Comment by Terala-Priyanka** [comment]
@folio-org/fe-tl-reviewers Please take a look, I need one more review

**Review by vashjs — APPROVED** [review]

**Pull request merged**